gateway-service/src/test/java/org/zowe/apiml/gateway/security/service/schema/source/AuthSourceTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.gateway.security.service.schema.source;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.zowe.apiml.security.common.token.TokenNotValidException;
+
+public class AuthSourceTest {
+    @Test
+    void testSource() {
```

Comment:
Could you please split this test with Given When Then pattern and nesting instead of having all the assertion in one test? It's a bit more complicated to understand what exactly is tested and also it then doesn't really provide the information about what it should do.

Reply:
Done, please review